boards/cy8ckit-050/include/board.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2014 INRIA
+ * Copyright (C) 2014 Freie UniversitÃ¤t Berlin
```

Comment:
somehow here github doesn't show the umlaut correctly, but it does above. Wrong charset in one of the files? (should be utf-8 everywhere...)